### DISCUSSION TITLE
Connection problems

### DISCUSSION THREAD

**Discussion by Cedi00123** [emsesp/EMS-ESP32#1777] [Q&A]
Hello,

I have a Issue with the BUS-Connection. I connect the Bus to the correct port but it have very much issues on the rx.

Can someone help me?

I'm using a esp32 "MH-ET Live 32" on a "https://github.com/rocksolidsr/esp-ems-board/tree/main" to a "Logamatic MC110"

<img width="833" alt="image" src="https://github.com/emsesp/EMS-ESP32/assets/115263422/293d5c0b-98c7-42da-bccf-0e8add35dcb8">
<img width="1094" alt="image" src="https://github.com/emsesp/EMS-ESP32/assets/115263422/853d6e23-669f-45a3-a0d0-f65efc1ad372">

**Comment by proddy** [comment]
It's 99% a wiring problem. Did you switch Tx/Rx? Is it compatible with the circuit board you built?

**Comment by mrfixit1** [comment]
Or the known power error with early ESP32s? Mine needed a 47uF cap from 3V3 to GND to solve a brownout issue, see https://docs.espressif.com/projects/esptool/en/latest/esp32s3/troubleshooting.html#insufficient-capacitance.
